src/scanning/Thumbnail/Thumbnail.jsx

```diff
@@ -0,0 +1,33 @@
+import "./Thumbnail.scss";
+import { getThumbnailAltAndLink, getThumbnailHeader } from "../scanning.js";
+
+export const Thumbnail = ({ name, ra, dec, url }) => {
+  const { alt, link } = getThumbnailAltAndLink(name, ra, dec);
+  return (
+    <div className={`thumbnail ${name}`}>
+      <div className="thumbnail-name">{getThumbnailHeader(name)}</div>
+      <div className="thumbnail-image">
+        <img
+          className="crosshairs"
+          src="https://preview.fritz.science/static/images/crosshairs.png"
```

Comment:
later on you'll probably want to integrate these pngs in your app itself. They are static images we don't update, so having it in the app to not rely on preview to be up would be good.

Reply:
I created issue #16 for this